Sources/StreamVideoSwiftUI/Utils/PaddingsConfig.swift

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A configuration object that defines padding values for different edges.
+///
+/// `PaddingsConfig` allows you to specify custom padding values for leading,
+/// trailing, top, and bottom edges, which can be reused across multiple UI components.
+public struct PaddingsConfig {
+    
+    /// The amount of padding on the leading edge.
+    public let leading: CGFloat
+    
+    /// The amount of padding on the trailing edge.
+    public let trailing: CGFloat
+    
+    /// The amount of padding on the top edge.
+    public let top: CGFloat
+    
+    /// The amount of padding on the bottom edge.
+    public let bottom: CGFloat
+    
+    /// Creates a new `PaddingsConfig` instance with the specified edge paddings.
+    ///
+    /// - Parameters:
+    ///   - leading: The padding for the leading edge. Defaults to `0`.
+    ///   - trailing: The padding for the trailing edge. Defaults to `0`.
+    ///   - top: The padding for the top edge. Defaults to `0`.
+    ///   - bottom: The padding for the bottom edge. Defaults to `0`.
+    public init(
+        leading: CGFloat = 0,
+        trailing: CGFloat = 0,
+        top: CGFloat = 0,
+        bottom: CGFloat = 0
+    ) {
+        self.leading = leading
+        self.trailing = trailing
+        self.top = top
+        self.bottom = bottom
+    }
+}
+
+extension PaddingsConfig {
+    
+    /// Padding configuration for the participant info view.
+    ///
+    /// - Leading: `6`
+    /// - Trailing: `6`
+    /// - Top: `2`
+    /// - Bottom: `2`
+    @MainActor public static let participantInfoView = PaddingsConfig(
```

Comment:
Why @MainActor? Can we simply make it nonisolated(unsafe)?

Reply:
all this stuff should be on the main actor

Reply:
Why is that? Those things aren't access concurrently and the @MainActor is currently messing our API

Reply:
they are only accessed from the main actor in any case

Reply:
The fact they are accessed only from MainThread doesn't require the @MainActor limitation.

Reply:
not needed now that we use edge insets